### DISCUSSION TITLE
Incomplete message even after completely watching the session.

### DISCUSSION THREAD

**Discussion by anikethhebbar** [github/india#152] [Students]
I've watched the session without skipping and fast forwarding.. yet the progress is incomplete.. anyone help me out

**Comment by Aditya-Bhate** [comment]
Please ensure you watch the video in its entirety at one go. A page refresh, increased playback speed or jumping ahead in the video will erase the progress.

**Reply by anikethhebbar** [reply]
I mentioned that.. that's why I'm asking why isn't it updating..? You've just copy pasted what's there in their website 😅

**Reply by AyushSenDazzler** [reply]
**Try watching it again ,I encountered same problem while completing industry **session****

**Comment by scmishra-cse** [comment]
> I've watched the session without skipping and fast forwarding.. yet the progress is incomplete.. anyone help me out

I have also watched the whole first video of industry sessions Digitization and Automation in BFSI video without a single break or pause still didn't mark as complete.
